pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -7,6 +7,8 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	rbac "k8s.io/kubernetes/pkg/apis/rbac"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
```

Comment:
> We already import this dependency into this package for the other use

Yes, the mistake will be more obvious moving forward.  I won't block on it, but it is a problem.